src/libecalc/fixtures/cases/consumer_system_v2/consumer_system_v2_dto.py

```diff
@@ -231,7 +239,14 @@
     consumes=ConsumptionType.FUEL,
     fuel=fuel,
     component_conditions=SystemComponentConditions(
-        crossover=[0, 1, 1],
+        crossover=[
+            SystemCrossoverStream(
+                from_component_id=generate_id("compressor2"), to_component_id=generate_id("compressor1")
```

Comment:
Using component_id as name is not something we use in core, but this interface isn't very nice when creating a crossover stream in python, in dtos the id is created automatically. We could do that for crossover stream also

Reply:
If we take the unique name one step further, to enforce unique names across installations as well..it would be easier? If we can...then we can skip that `generate_id()` ...

Reply:
We have unique names across installations, so id is just a hash of name, we could use name, but if we do that in core, then what is the point of the id? But it would make things easier, so I think we can use name as id.